connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -291,15 +307,16 @@ object SparkConnectService extends Logging {
   }
 
   // Simple builder for creating the cache of Sessions.
-  private def cacheBuilder(cacheSize: Int, timeoutSeconds: Int): CacheBuilder[Object, Object] = {
+  private[service] def cacheBuilder(
```

Comment:
Is the vis change needed?

Reply:
Yes, SessionHolder needs to access cacheBuilder

Reply:
Why do we have the crossing concerns here?

Reply:
I will isolate them then